### PR TITLE
Json encoding

### PR DESCRIPTION
**Pull request by justin808**

### CODE REVIEW

**Comment by justin808** [comment]
Note, second commit is all linting issues.

**Comment by justin808** [comment]
Fixes #47

**Comment by dylangrafmyre** [comment]
@justin808 awsome1 :+1:

File: app/helpers/react_on_rails_helper.rb [hunk 1]
```diff
@@ -94,11 +94,7 @@ def react_component(component_name, props = {}, options = {})
   end
 
   def sanitized_props_string(props)
-    props_string = props.is_a?(String) ? props : props.to_json
-    # Need to find something like this:
-    # ActiveSupport::JSON.encode(props_string)
-    # Next PR
-    props_string
+    props.is_a?(String) ? json_escape(props) : props.to_json
```
Comment (alex35mil) [inline app/helpers/react_on_rails_helper.rb:97]: :+1:

**Pull request merged**